backend/app/project/routes.py

```diff
@@ -0,0 +1,8 @@
+from flask import Flask, jsonify, request
+from . import project
+
+# get all projects
+@project.route("", methods=["GET"])
+def get_projects():
+    projects = project.query.all()
+    return jsonify(project.serialize_list(projects))

```

Comment:
This `serialize_list` method doesn't actually exist under the project model. Could you add it in order for this to work?

Something similar to this
https://github.com/celinehuang/kom_get_your_bucha/blob/master/backend/app/models.py#L69L77